<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customGUIDemoClass</class>
 <widget class="QMainWindow" name="customGUIDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>customGUIDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QWidget" name="widget" native="true"/>
    </item>
    <item row="7" column="0">
     <widget class="QWidget" name="widget_2" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="m_pbLogin_Out">
      <property name="text">
       <string>登陆</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>PushButton1</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>PushButton2</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>PushButton3</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="m_pbUserMgr">
      <property name="text">
       <string>用户管理</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="customGUIDemo.qrc"/>
 </resources>
 <connections/>
</ui>
